src/askem_beaker/contexts/mira_model_edit/context.py

```diff
@@ -34,11 +34,9 @@ def __init__(self, beaker_kernel: "LLMKernel", subkernel: "BaseSubkernel", confi
         self.reset()
         self.auth = get_auth()
         super().__init__(beaker_kernel, subkernel, self.agent_cls, config)
-
-    def reset(self):
-        pass
-        
+    
     async def setup(self, config, parent_header):
+        logger.error(f"performing setup...")
```

Comment:
Flagging this as an error might be a little excessive

Reply:
LOL Fair I just grabbed the same status in the newest context I could find model_config_edit 
ill update this